docs/rfcs/managed.md

```diff
@@ -0,0 +1,150 @@
+# Managed CRD
+
+**Status:** *Draft*
+
+**Author:** Antoine Toulme (@atoulme)
+
+**Date:** 2025-03-20
+
+## Objective
+
+Condense all best practices and deployment patterns of OpenTelemetry components into a single custom resource
+that covers the use cases fulfilled by the OpenTelemetry helm charts.
+
+## Summary
+This request for comment aims to establish a new approach to management of resources in Kubernetes clusters,
+with the explicit goal of taking over the whole cluster, and making decisions
+on how to best instrument all resources in the cluster on behalf of the administrator based on best practices,
+OpenTelemetry standards and semantic conventions.
+
+The operator project offers custom resources (CR) to allow Kubernetes users/administrators to define OpenTelemetry components
+in their Kubernetes environments. Customers however do not want to manage components themselves, especially if they need
+to think about component versioning and upgrade patterns, and would rather have an out-of-the-box experience that 
+requires little insight and gets them to a working state effortlessly.
+
+## Goals and non-goals
+
+### Give a comprehensive, out-of the box experience of OpenTelemetry for a first-time user
+The goal is to delight first-time customers by giving them a complete view of what is possible with OpenTelemetry.
+
+### Do not offer a way to turn on and off specific features
+We mean to offer a specific functional coverage of a specific use case that offers best practices of OpenTelemetry.
+
+We don't allow to toggle features in and out specifically because we want to avoid increasing the need for functional test coverage.
```

Comment:
```suggestion
We don't allow toggling features specifically because we want to avoid increasing the need for functional test coverage.
```

Reply:
I'm not sure I love this reason to not allow toggling. I think it would be more accurate to say "this CR is meant as an introduction to the features OpenTelemetry has and is not meant to fulfill every use case." This then flows much better into the next paragraph.